debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -776,7 +776,7 @@ protected void execute() {
                             if (!isRunning()) {
                                 break;
                             }
-                            readTableSchema(sql, mysql, schema, source, tableId.schema(), tableId);
+                            readTableSchema(sql, mysql, schema, source, tableId.catalog(), tableId);
```

Comment:
Interesting; we use `schema()` also elsewhere (line 485); looks like this should be adjusted, too?

Reply:
@gunnarmorling Updated, thanks, lokks like a nasty long-standing bug